sdk/iot/core/inc/az_iot_core.h

```diff
@@ -21,12 +21,16 @@
 /**
  * @brief Azure IoT service MQTT connection properties.
  *
+ * @note You MUST NOT use any symbols (macros, functions, structures, enums, etc.)
```

Comment:
Consider moving this `@note` to the `@file` comment block above, like other places. I don't think it belongs to this particular enum.